sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -8,6 +8,7 @@ export enum Role {
   AESTManualTriggerReassessment = "aest-manual-trigger-reassessment",
   AESTReports = "aest-reports",
   AESTCreateInstitution = "aest-create-institution",
+  AESTEditCASSupplierInfo = "aest-edit-cas-supplier-info",
```

Comment:
Are we using this?

Reply:
I just added this as when I merged in from main the tests I hadn't touched started failing due to this being missing.

Reply:
Please remove anything which is not in the scope of the PR. If the tests fail due to AEST role, it should be fine.

Reply:
removed